### PR TITLE
New Management Library for Security Insights (Sentinel)

### PR DESCRIPTION
**Pull request by dicolanl**

### CODE REVIEW

**Review by markcowl — CHANGES_REQUESTED** [review]
The generation script creates a file in the /eng folder with metadata about the swagger spec - please include this file in the PR.  Also, you will need to add your project here: https://github.com/Azure/azure-sdk-for-net/blob/master/eng/pipelines/mgmt.yml  so that tests can run as part of ci.

**Comment by dicolanl** [comment]
@markcowl  the mgmt.yml file has securityinsights.
image

I added the securityinsights_resource_manager.txt

**Comment by markcowl** [comment]
@dicolanl There is a reported error in project references - restarted in case this is transient

File: eng/pipelines/mgmt.yml [hunk 1]
```diff
@@ -14,8 +14,8 @@ pr:
     - sdk/appplatform/Microsoft.Azure.Management.AppPlatform
     - sdk/attestation/Microsoft.Azure.Management.Attestation
     - sdk/authorization/Microsoft.Azure.Management.Authorization
-    - sdk/automation/Microsoft.Azure.Management.Automation
     - sdk/automanage/Microsoft.Azure.Management.Automanage
+    - sdk/automation/Microsoft.Azure.Management.Automation
     - sdk/avs/Microsoft.Azure.Management.Avs
```
Comment (markcowl) [inline eng/pipelines/mgmt.yml:19]: Please don't make other changes to this file.  Please just add your change (note that you might need to pull the latest from master if you did not make any changes).

**Review by markcowl — CHANGES_REQUESTED** [review]

**Comment by markcowl** [comment]
@dicolanl  A few too many changes to the management yml, otherwise, this is ready to merge.

File: eng/pipelines/mgmt.yml (hunk 1 above)
Comment (dicolanl) [reply eng/pipelines/mgmt.yml]: I just downloaded the yml from master.  copied over the version in my branch and added security insights.

**Review by markcowl — APPROVED** [review]

**Pull request merged**